TestProject/Models/SetRatingModel.cs

```diff
@@ -1,11 +1,10 @@
-﻿using TestProject.DAL.Enums;
+﻿using TestProject.BL.Enums;
```

Comment:
Better than DAL, but the initial issue remains - it's not a rating. In fact it's just button that pressed. So `Plus` actually performs the opposite action in some cases. Separate the button pressed and action performed. Let's discuss it as well.